src/test/java/no/sikt/nva/pubchannels/channelregistrycache/db/service/CacheServiceDynamoDbSetup.java

```diff
@@ -0,0 +1,76 @@
+package no.sikt.nva.pubchannels.channelregistrycache.db.service;
+
+import static no.sikt.nva.pubchannels.channelregistrycache.db.model.ChannelRegistryCacheDao.PRIMARY_KEY;
+import static no.sikt.nva.pubchannels.channelregistrycache.db.model.ChannelRegistryCacheDao.SORT_KEY;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.dynamodbv2.local.embedded.DynamoDBEmbedded;
+import java.nio.file.Path;
+import no.sikt.nva.pubchannels.channelregistrycache.ChannelRegistryCacheConfig;
+import no.sikt.nva.pubchannels.channelregistrycache.ChannelRegistryCsvLoader;
+import no.unit.nva.s3.S3Driver;
+import no.unit.nva.stubs.FakeS3Client;
+import nva.commons.core.Environment;
+import nva.commons.core.ioutils.IoUtils;
+import nva.commons.core.paths.UnixPath;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeDefinition;
+import software.amazon.awssdk.services.dynamodb.model.CreateTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.KeySchemaElement;
+import software.amazon.awssdk.services.dynamodb.model.KeyType;
+import software.amazon.awssdk.services.dynamodb.model.ProvisionedThroughput;
+import software.amazon.awssdk.services.dynamodb.model.ScalarAttributeType;
+
+public class CacheServiceDynamoDbSetup {
+
+    private DynamoDbClient client;
+
+    public void setup(){
+        this.client = DynamoDBEmbedded.create().dynamoDbClient();
+        createTable(new Environment().readEnv("TABLE_NAME"));
+    }
+
+    public void loadCache() {
+        var csv = IoUtils.stringFromResources(Path.of("cache.csv"));
+        var s3Client = new FakeS3Client();
+        var s3Driver = new S3Driver(s3Client, ChannelRegistryCacheConfig.CACHE_BUCKET);
```

Comment:
Feel like I'm missing something…is this a test context?

Reply:
This is a test setup yes :)